### ISSUE TITLE
Not properly clean the cache on composer update

### ISSUE THREAD

**Issue by peter-gribanov** [sensiolabs/SensioDistributionBundle#139]
composer.json

``` json
"post-update-cmd": [
    "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
    "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
    "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
    "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
    "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile"
],
```

Handler `clearCache` is called after the installation and upgrade project and calls the command `cache:clear --no-warmup`. This command deletes the cache folder for the current environment. When updating from the composer environment is not specified and the environment is taken by default `dev`. As a result, the cache of production is not cleared, which leads to serious consequences.

For example, installing\removing packages in `AppKernel` and they will not work until the cache is not updated.

That is, after the upgrade project need to update the cache for all environments. With the latest changes necessary to consider the option `symfony-cache-warmup`.

Simply deleting cache `prod` environment is not always suitable. If the package has been removed which was an event listener console we get a fatal error.
`cache:clear --env=prod --no-warmup`

**Comment by vishalraj82** [comment]
With the release of Symfony 3, the console script has been moved from app folder to bin.
Have a look at file - vendor/sensio/distribution-bundle/Composer/ScriptHandler.php, line 452
The return statement must be changed to "return $options['symfony-bin-dir'];"

**Comment by aschempp** [comment]
I would suggest an option for the Symfony environment that should be cleared. If the script handler can handle `--env=prod` then someone in need of this can simply add the script handler multiple times to the `composer.json`

**Comment by HeahDude** [comment]
This can be closed by #258.

**Comment by fabpot** [comment]
Closing this pull request as this bundle is now deprecated. Symfony Flex replaces this bundle.
